### PR TITLE
Function filter country

### PR DESCRIPTION
**Pull request by esaschenck**
Successfully wrote functions to guide the user through basic R setup for data.

### CODE REVIEW

File: R/data_wrangling_functions.R [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#' Check to see if the user input is correct
+#'
+#' @param typed
+#' @param correct
+#'
+#' @return
+#' @export
+#'
+#' @examples
+check_if_correct <- function(typed, correct){
```
Comment (margotdv04) [inline R/data_wrangling_functions.R:10]: good job `computer_font`

File: R/data_wrangling_functions.R [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+#' See Data In Console
+#'
+#' @return
+#' @export
+#'
+#' @examples
+see_data_console <- function(){
+  cat("Now you will learn to see the data! \n")
```
Comment (margotdv04) [inline R/data_wrangling_functions.R]:
```suggestion
  cat("Now you will learn to see the data!!! \n")
```

**Review by margotdv04 — CHANGES_REQUESTED** [review]
here are my comments

File: R/data_wrangling_functions.R (hunk 1 above)
Comment (esaschenck) [reply R/data_wrangling_functions.R:10]: thanks!

**Review by margotdv04 — APPROVED** [review]

**Pull request merged**